extensions/hibernate-orm/deployment/src/main/resources/dev-ui/hibernate-orm-hql-console.js

```diff
@@ -4,12 +4,14 @@ import {JsonRpc} from 'jsonrpc';
 import '@vaadin/icon';
 import '@vaadin/button';
 import '@vaadin/combo-box';
-import '@vaadin/grid';
+import '@vaadin/text-field';
+import '@vaadin/text-area';
 import '@vaadin/progress-bar';
 import '@vaadin/tabs';
 import '@vaadin/tabsheet';
-import {columnBodyRenderer} from '@vaadin/grid/lit.js';
 import {notifier} from 'notifier';
+import { assistantState } from 'assistant-state';
+import 'qui-assistant-warning';
 
 export class HibernateOrmHqlConsoleComponent extends QwcHotReloadElement {
```

Comment:
I missed this in my initial review. You also need to observe for state changes. So:

`import { observeState } from 'lit-element-state';`

And then :

`export class HibernateOrmHqlConsoleComponent extends observeState(QwcHotReloadElement) {`